ReactiveArchitecture/Classes/nowplaying/controller/ServiceControllerImpl.swift

```diff
@@ -45,18 +45,18 @@ class ServiceControllerImpl : ServiceController {
     }
 
     func getNowPlaying(pageNumber: Int) -> Observable<NowPlayingInfo> {
-        DDLogInfo("Thread name: " + Thread.current.debugDescription + " Get NowPlaying for Page #" + pageNumber.description);
+        DDLogInfo("Thread name: " + Thread.current.debugDescription + " Get NowPlaying for Page #" + pageNumber.description)
        
-        var mapToSend:Dictionary<String, Int> = [String: Int]()
+        var mapToSend: Dictionary<String, Int> = [String: Int]()
         mapToSend["page"] = pageNumber
         
         /*
          Notes - Load data from web on scheduler thread. Translate the web response to our
          internal business response on computation thread. Return observable.
          */
         return serviceApi.nowPlaying(apiKey: apiKey, query: mapToSend)
-            .flatMap{ (serviceResponse: ServiceResponse) -> Observable<NowPlayingInfo> in
-                return TranslateNowPlayingSubscriptionFunc.init(imageUrlPath: self.imageUrlPath).apply(serviceResonse: serviceResponse)
+            .flatMap { (serviceResponse: ServiceResponse) -> Observable<NowPlayingInfo> in
+                TranslateNowPlayingSubscriptionFunc.init(imageUrlPath: self.imageUrlPath).apply(serviceResonse: serviceResponse)
```

Comment:
I'll probably add the return back in for learning. put some kind of note in the readme...why they are there lol